src/components/Graphs/LineGraph.js

```diff
@@ -1,6 +1,58 @@
-import React from "react";
-import {LineGraphStyle} from "./LineGraphStyle";
+import React, { PureComponent } from 'react';
+import { LineGraphStyle } from './LineGraphStyle';
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend
+} from 'recharts';
 
-export const LineGraph = () => {
-  return <LineGraphStyle> This is a div for Line Graph </LineGraphStyle>;
-};
+const data = [
+  {
+    name: 'cohort-6',
+    count: 43
+  },
+  {
+    name: 'cohort-8',
+    count: 43
+  },
+  {
+    name: 'cohort-7',
+    count: 44
+  }
+];
+
+export default class LineGraph extends PureComponent {
```

Comment:
Does this have to be a PureComponent because of Rechart?

Reply:
Found this: https://codeburst.io/when-to-use-component-or-purecomponent-a60cfad01a81 

It's interesting, I think that using the pure component here does add value; I think the graph will not be re-rendered each time that something changes in state or props from HomePage. hope I understood it correctly.